docker/pypi/gfal/Dockerfile.alma9

```diff
@@ -0,0 +1,23 @@
+FROM registry.cern.ch/cmsweb/dmwm-base:alma9-py3.12-20250515
+
+WORKDIR /tmp
```

Comment:
I am inclined to say that we could rename this dockerfile as:
* the current debian/conda based to be named as Dockerfile.debian
* and this file to Dockerfile

What do you think?